### DISCUSSION TITLE
Hackathon project ideas

### DISCUSSION THREAD

**Discussion by groenborg** [devx-cafe/no-code-hackathon-okt6#2] [Ideas]
## Ideas for projects: 

- [**Baby tracker**](https://dribbble.com/shots/17067181-Baby-tracker-app-design)
- [**Task management**](https://dribbble.com/shots/19516907-ERP-SaaS-Web-App)
- [**Finance tracker**](https://www.behance.net/gallery/152458779/Financial-App-for-money-management)
- [**Time tracker**](https://dribbble.com/shots/11616683-Mobile-Time-Tracking-App)
- [**Plant shop**](https://dribbble.com/shots/13880920-Plant-Shop/attachments/5488772?mode=media)
- [**Webshop**](https://dribbble.com/shots/19210222-Polestar-Additionals-Ecommerce)

## Learning
- [Webflow academy](https://university.webflow.com) 
- [AppSheet 101](https://support.google.com/appsheet/answer/10099795)

**Comment by groenborg** [comment]
- Vinino pandang
- Finance manager + task tracker

**Comment by orskoven** [comment]
Tak for et spændende og øjenåbnende arrangement!
🙏

**Comment by vskytte1** [comment]
Tak for et super inspirerende arrangement i dag. Glæder mig til at følge med i kommende NoCode-projekter…
